src/Service/VendorService/TheMovieDatabase/TheMovieDatabaseSearchService.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+/**
+ * @file
+ * Handle search at the data well to utilize hasCover relations.
+ */
+
+namespace App\Service\VendorService\TheMovieDatabase;
+
+use App\Exception\DataWellVendorException;
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\RequestOptions;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+/**
+ * Class SearchService.
+ */
+class TheMovieDatabaseSearchService
```

Comment:
Why this special search for this function... we should re-use the openplatform SearchService or make a base and create to extension on that so all openplatform search lives in the `App\Service\OpenPlatform` namespace.

It will make updating openplatform to new version in one place.

Reply:
Will add todo